### DISCUSSION TITLE
SOLWEIG - no data output if CDSM input

### DISCUSSION THREAD

**Discussion by elidamd** [UMEP-dev/UMEP#390] [General]
Hi,
I am testing SOLWEIG inputs, once I flag the "use vegetation scheme" option and CDSM input, the output is empty. If I run the same process using only DSM and landcover, the output is properly given.
I would like to ask for some tips on how to find out the error (what is wrong with the files). 
CDSM file was generated from point-type shp of trees in the Tree Generator tool. This file seems to be okay, as I tested with the Daily Shadow Pattern tool.

OS: Windows 10
QGIS Version: 3.22.2
UMEP Version: 3.19.3
SuPy Version: 2022.2.17

Thanks a lot!

**Comment by biglimp** [comment]
Do you get an error message when you run SOLWEIG with vegetation scheme?

**Reply by elidamd** [reply]
I don't get any errors.. Instead, an output (Tmrt.tif) with no data (z values).

**Reply by biglimp** [reply]
Can you attach the CDSM please, as well as your RunInfo file that should be located in your output folder.

**Reply by elidamd** [reply]
[cdsm.zip](https://github.com/UMEP-dev/UMEP/files/8408679/cdsm.zip)
I send the files attached here, thanks!

**Reply by biglimp** [reply]
Your cdsm looks fine, also the info in RunInfo. hm... could you maybe also attach the rest of the data (DSM, DEM, Land cover and SVF.zip) so that I can try to run your data.

**Comment by elidamd** [comment]
unfortunately, I cannot share the other files. I will perform other tests with new files to share with you. Thanks!

**Reply by elidamd** [reply]
I worked on a very small scale of my project to replicate the error and share the files, but now adding the cdsm option worked well. 
I noticed that in the Solweig directory (...python\plugins\UMEP\SOLWEIG), there are many files created during the process (svfveg.tif and similar) and while testing the process I had error in some of these files. Once I removed them in this main folder, the process was concluded. So I will test my project again, maybe this was the issue. 
Thanks for the support!
